src/pages/blog/airship-update-july-2020.md

```diff
@@ -0,0 +1,138 @@
+---
+templateKey: blog-post
+title: Airship Update - July 2020
+author: Alexander Hughes
+date: 2020-07-11T12:00:00.000Z
+category:
+  - id: category-C98iZYrE1
+    label: Airship 2.0
+---
+
+What follows is the Airship Community Update for July 2020, brought to you by the Airship Technical Committee.
+<!-- more -->
+
+## **SPOTLIGHT ON...**
+
+- Last month the community elected the 2020-2021 Technical Committee. For more information, check out this blog
+  [_post_](https://www.airshipit.org/blog/2020-airship-tc-election/).
+- Ironic published a White Paper on Baremetal with input from multiple projects, including members of Airship. Check it
+  out [_here_](PLACEHOLDER LINK EXPECTED TO BE PUBLISHED WEEK OF 13 JULY).
+
+<br>
+
+## **AIRSHIP BLOG**
+
+The Airship blog is a great way to keep up with what's going on in the community. The Airship community publishes
+[_blog posts_](https://www.airshipit.org/blog/) regularly, including the new Airship 2.0 Blog series. These blog posts
+introduce the changes from Airship 1.0 to Airship 2.0, highlight new features, and detail each component's evolution.
+The first six Airship 2.0 Blog posts are already available, and illustrate the design thoughts on the road to the alpha
+milestone:
+
+* [_Airship Blog Series 1 - Evolution Towards 2.0_](
+  https://www.airshipit.org/blog/airship-blog-series-1-evolution-towards-2.0.html)
+* [_Airship Blog Series 2 - An Educated Evolution_](
+  https://www.airshipit.org/blog/airship-blog-series-2-an-educated-evolution.html)
+* [_Airship Blog Series 3 - Airship 2.0 Architecture High Level_](
+  https://www.airshipit.org/blog/airship-blog-series-3-airship-2.0-architecture-high-level.html)
+* [_Airship Blog Series 4 - Shipyard - an Evolution of the Front Door_](
+  https://www.airshipit.org/blog/airship-blog-series-4-shipyard-an-evolution-of-the-front-door.html)
+* [_Airship Blog Series 5 - Drydock and Its Relationship to Cluster API_](
+  https://www.airshipit.org/blog/airship-blog-series-5-drydock-and-its-relationship-to-cluster-api.html)
+* [_Airship Blog Series 6 - Armada Growing Pains_](
+  https://www.airshipit.org/blog/airship-blog-series-6-armada-growing-pains.html)
+
+<br>
+
+Stay tuned for more exciting news on the Airship blog series as Airship 2.0 progresses through the beta milestone and
```

Comment:
Have we documented anywhere the value add of Beta vs Alpha?  If so, it could be good to link to that over the word beta.

Reply:
Not yet, Alan and Rodolfo are wrapping up a blog now that summarizes Alpha, the next one in the series will define Beta.